rtengine/iptransform.cc

```diff
@@ -955,6 +945,10 @@ void ImProcFunctions::transformGeneral(bool highQuality, Imagefloat *original, I
                 y_d *= maxRadius * vpcospt / (maxRadius - y_d * vptanpt);
             }
 
+            if (enableLCPDist) {
+                pLCPMap->correctDistortion(x_d, y_d, w2, h2, 1); // must be first transform
```

Comment:
The comment `must be first transform` is confusing now, as it's not the first trasnform anymore ;-)
Everything else looks good to me. :+1:  for merge

Reply:
Comment removed.